http/README.md

```diff
@@ -14,6 +14,7 @@ perform an HTTP DELETE request, returning a response
 | `headers` | `dict` | optional. dictionary of headers to add to request |
 | `body` | `string` | optional. raw string body to provide to the request |
 | `form_body` | `dict` | optional. dict of values that will be encoded as form data |
+| `form_encoding` | `string` | optional. `multipart/form-data` (default) or `application/x-www-form-urlencoded` |
```

Comment:
this file is generated automatically by running [outline](https://github.com/b5/outline) against `http/doc.go`. If you could make these changes there instead I'd be grateful. An example of adding this [to the `post` method documentation](https://github.com/qri-io/starlib/blob/1364b0c91ab2f33e7443baae52334202009a52b1/http/doc.go#L31-L45) would look like this:

```
post(url,params={},headers={},body="",form_body={},json_body={},auth=()) response
        perform an HTTP POST request, returning a response
        params:
          url string
            url to request
          headers dict
            optional. dictionary of headers to add to request
          body string
            optional. raw string body to provide to the request
          form_body dict
            optional. dict of values that will be encoded as form data
          form_encoding string
            optional. `multipart/form-data` (default) or `application/x-www-form-urlencoded`
```

We'll also need to document the `postForm` function this PR adds. We can leverage go's documentation for language.

Reply:
ps: outline is whitespace sensitive